README.md

```diff
@@ -42,10 +42,24 @@ Usage of ./server:
     	print version number and build date
 ```
 
-If you are working on the sqlite migrations, html templates or website assets, build the server with `go build -tags dev`.
-This way it won't use the assets that are embedded in the binary but read them directly from the local filesystem.
+If you want to view the development server in your browser:
+```sh
+cd cmd/server && go build -tags dev && ./server
+# and visit http://localhost:3000
+```
+
+This can be useful if you are working on:
+* the sqlite migrations, 
+* html templates, 
+* or website assets
 
-Once you are done with your changes run `go generate` in the changed packages to update them.
+This way the build won't use the assets embedded in the binary, but instead read them directly from the local filesystem.
+
+Once you are done with your changes run and want to update the embedded assets:
+```sh
+go generate
+```
+**Note**: you need to run generate in each changed package.
```

Comment:
this can be `go generate ./...` from the root folder of the repo.

the only thing that is a big quirky is #52 - for development you need `go generate -tags dev` to get the correct tailwind css.